elements/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -228,6 +210,22 @@ export class MultisignatureTransaction extends BaseTransaction {
 		return errors;
 	}
 
+	public validateFee(): TransactionError | undefined {
+		const expectedFee = new BigNum(MultisignatureTransaction.FEE).mul(
```

Comment:
Multisig transaction has a constant fee irrespective of keys group length. Did we update the fee calculation already?

Reply:
I think the calculation is correct per current protocol for register multi-signature transaction

Reply:
My bad. Looked at the protocol definition again.